ironfish/src/rpc/routes/accounts/getNotes.ts

```diff
@@ -18,44 +18,44 @@ export type GetAccountNotesResponse = {
   }[]
 }
 
-export const GetAccountNotesRequestSchema: yup.ObjectSchema<GetAccountNotesRequest> = yup
-  .object({
-    account: yup.string().strip(true),
-  })
-  .defined()
-
-export const GetAccountNotesResponseSchema: yup.ObjectSchema<GetAccountNotesResponse> = yup
-  .object({
-    account: yup.string().defined(),
-    notes: yup
-      .array(
-        yup
-          .object({
-            owner: yup.boolean().defined(),
-            amount: yup.number().defined(),
-            memo: yup.string().trim().defined(),
-            transactionHash: yup.string().defined(),
-            spent: yup.boolean(),
-          })
-          .defined(),
-      )
-      .defined(),
-  })
-  .defined()
-
-router.register<typeof GetAccountNotesRequestSchema, GetAccountNotesResponse>(
-  `${ApiNamespace.account}/getAccountNotes`,
-  GetAccountNotesRequestSchema,
+export const GetAccountNotesStreamRequestSchema: yup.ObjectSchema<GetAccountNotesStreamRequest> =
+  yup
+    .object({
+      account: yup.string().strip(true),
+    })
+    .defined()
+
+export const GetAccountNotesStreamResponseSchema: yup.ObjectSchema<GetAccountNotesStreamResponse> =
+  yup
+    .object({
+      account: yup.string().defined(),
+      notes: yup
+        .array(
+          yup
+            .object({
+              owner: yup.boolean().defined(),
+              amount: yup.number().defined(),
+              memo: yup.string().trim().defined(),
+              transactionHash: yup.string().defined(),
+              spent: yup.boolean(),
+            })
+            .defined(),
+        )
+        .defined(),
+    })
+    .defined()
+
+router.register<typeof GetAccountNotesStreamRequestSchema, GetAccountNotesStreamResponse>(
+  `${ApiNamespace.account}/getAccountNotesStream`,
+  GetAccountNotesStreamRequestSchema,
   async (request, node): Promise<void> => {
     const account = getAccount(node, request.data.account)
 
-    const responseNotes = []
-
     for await (const { transaction } of account.getTransactions()) {
       const notes = await getTransactionNotes(account, transaction)
-      responseNotes.push(...notes)
+      request.stream({ account: account.displayName, notes: [...notes] })
```

Comment:
The stream output here should be each note, it looks like it's outputting a stream of chunk of notes per transaction.